packages/docs/docs/website/sidebars.json

```diff
@@ -2,8 +2,9 @@
   "docs": {
     "OVERVIEW": [
       "start",
-      "new_2",
-      "new_2.2"
+      "new_2.3",
```

Comment:
Can we apply versioning to the sidebar so we don't accumulate old `what's new` sections?. I expect to see only the  `What's new in ZeppelinOS 2.3` when I'm in looking at the `2.3` version of the docs and not the `What's new in ZeppelinOS 2.2` or `What's new in ZeppelinOS 2.0`. 
Not a blocker though, we can change it before releasing the new docs!

Reply:
Good idea! Pushed in c6943285e65c6f892b74c0ea5e41b88a7754b42c.